github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/dsl/context/GitHubPRTriggerDslContext.java

```diff
@@ -33,6 +35,14 @@ public void setPreStatus() {
         setPreStatus = true;
     }
 
+    public void setCancelQueued() {
```

Comment:
just `cancelQueued` as of `set` only to reflect that its setter. No need to reflect same in dsl